lua/nvim-treesitter/query.lua

```diff
@@ -111,9 +111,13 @@ function M.has_query_files(lang, query_name)
   return false
 end
 
-function M.get_query(lang, query_name)
+function M.get_query_string(lang, query_name)
```

Comment:
exposing this could be handy for some situations. I'm using it so see what queries are used for which language,

Reply:
Why do you need to know that ? Why not using `has_*`functions ?

Reply:
Just docgen. Which subset of the  queries are available in which language. Has a languages for instance the `@class.outer` textobject

Reply:
Then you can use `query.captures` with and already parsed query, this is the table of defined captures within the query.

Reply:
But then I need the parsers available. But would also be possible to compile them beforehand. Right now it's a plain find on a string.

Reply:
New version uses `query.captures` but it requires compiling the parsers beforehand.